py3dtilers/Common/README.md

```diff
@@ -178,6 +178,22 @@ You can skip leaf tiles and their parents geometric errors by writing a non nume
 tileset-reader --paths <tileset_path> --geometric_error x x 100  # Set root tiles GE to 100
 ```
 
+### Kd-tree max
+
+| Tiler | |
+| --- | --- |
+| CityTiler | :heavy_check_mark: |
+| ObjTiler | :heavy_check_mark: |
+| GeojsonTiler | :heavy_check_mark: |
+| IfcTiler | :heavy_check_mark: |
+| TilesetTiler | :heavy_check_mark: |
+
+`--kd_tree_max` allows to choose the maximum number of features in each tile when the features are distributed by a kd-tree. The flag must be followed by an __integer__. By default, each tile contains a maximum of 500 features.
+
```

Comment:
Maybe the note concerning the Texture case (When using the CityTiler with texture, this default value is divided by 20 to reduce the size of each image atlas...) should be recalled in the user docs ? 
Is this the correct place, or is it better to place it below the --texture  section of the documentation ?